.gitignore

```diff
@@ -23,4 +23,8 @@ node_modules
 yarn-error.log
 
 # IntelliJ
-.idea/
\ No newline at end of file
+.idea/
+
+# DartSass
```

Comment:
Very minor suggestion but could you please change `# DartSass` to `# Sass` or `# Dart Sass`.